tools/bots/flutter/analyze_flutter_packages.sh

```diff
@@ -26,15 +26,15 @@ git clone --single-branch -vv https://github.com/flutter/flutter
 export PATH="$PATH":"$tmpdir/flutter/bin"
 flutter --version
 
-# get the flutter/plugins repo
-git clone --single-branch -vv https://github.com/flutter/plugins
-cd plugins
+# get the flutter/packages repo
+git clone --single-branch -vv https://github.com/flutter/packages
+cd packages
 
 # validate the tool's source
 (cd script/tool; dart pub get)
 (cd script/tool; dart analyze --fatal-infos)
 
 # Invoke the repo's analysis script.
-./script/tool_runner.sh analyze \
+dart run script/tool/bin/flutter_plugin_tools.dart analyze \
```

Comment:
If we have any issues with this in the future, I'll create a custom wrapper called something like `script/out_of_repo_analysis.sh`, and update here and flutter/flutter to use it.